src/libs/httpserver/indexGenerator.js

```diff
@@ -67,10 +69,18 @@ const isSlugInAllowlist = currentSlug =>
       (slug === currentSlug && platform === Platform.OS)
   )
 
+const isSlugInBlocklist = currentSlug =>
+  slugBlockList.some(
+    ({ slug, platform }) =>
+      (slug === 'ALL' && platform === Platform.OS) ||
```

Comment:
Note that we implemented the "block" mechanism in the past: https://github.com/cozy/cozy-flagship-app/commit/9fcf6c0dc6a12c0e3758e41e8fd7fdc89e91e9a2

We chose to replace it by an "allow" list to fit the following logic: we should first fully test an app before allowing it in this mechanism
Put another way: we chose "opt-in" over "opt-out"

As today we have a new logic that need to implement the "opt-out" version back, then I would do the following:
- Add in the comments that we should priorize the use of the "allow" list over the "block" list
- I would simplify this condition as we should never have `slug === 'ALL'` in the "block" list (today we need at least `home` to be served on all platform and we don't want to disable it by mistake)

Reply:
Thank you for the context 👍